ovirt_imageio/client/_options.py

```diff
@@ -267,11 +268,14 @@ def _merge_config(self, args):
 
             # Read and validate value from config file.
             if config.has_option(args.config, option.name):
-                raw_value = config.get(args.config, option.name)
-                try:
-                    value = option.type(raw_value)
-                except ValueError as e:
-                    self._parser.error(e)
+                if option.action is StoreFalseOrNoneAction:
+                    value = config.getboolean(args.config, option.name)
```

Comment:
This is a bit ugly, depending on the type of the action. Can we use the type of the
default instead?

    if type(option.default) is bool:

But the best would be if the common code works for everything:

    value = option.type(raw_value)

And we implement converting from string to bool in our code instead of using
config.getboolean().

Reply:
Yeah, I was not convinced by this either. I tried with nested `try...except`, but the checks were misbehaving.

I guess that I will go back to the type doing the conversion.